py3status/modules/sysdata.py

```diff
@@ -9,10 +9,8 @@
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
     mem_unit: the unit of memory to use in report, case insensitive.
         ['dynamic', 'KiB', 'MiB', 'GiB'] (default 'GiB')
-    padding: length of space padding to use on the left
-        (default 0)
-    precision: precision of values
-        (default 2)
+    temp_unit: unit used for measuring the temperature
+        (default '°C')
```

Comment:
I don't like this setting.  All it is doing is creating a placeholder which feels like adding no value.

I think it would be better to have temp_unit actually be `C` or `F` for any crazy people who want Fahrenheit (`sensors -f`).  The placeholder would then become `°C` or `°F`.